openhcl/underhill_core/src/worker.rs

```diff
@@ -1603,6 +1603,7 @@ async fn new_underhill_vm(
         secure_boot: dps.general.secure_boot_enabled,
         tpm_enabled: dps.general.tpm_enabled,
         tpm_persisted: !dps.general.suppress_attestation.unwrap_or(false),
+        azure_filtered_devices_allowed: with_vmbus_relay && isolation.is_isolated(),
```

Comment:
>with_vmbus_relay && isolation.is_isolated(), [](http://example.com/codeflow?start=40&length=44)

Should this also && vpci_boot_enabled?

Either that, or let's plumb an additional dps bit for this.  The latter is only needed if we want MANA and not NVMe, which may just not be an interesting scenario